logger/src/logger.cpp

```diff
@@ -56,4 +57,20 @@ void logMessage(const std::string& message,
     logMessage(Loglevel::INFO, message, location);
 }
 
+void fetchNlogs(size_t noofLogs)
+{
+    std::fstream logfile("/tmp/logdata.txt", std::ios::trunc | std::ios::out);
+    if (noofLogs > circularBuffer.size())
+    {
+        noofLogs = circularBuffer.size();
+    }
+    auto iterator = circularBuffer.begin();
+    std::advance(iterator, circularBuffer.size() - noofLogs);
+    while (iterator != circularBuffer.end())
```

Comment:
Add empty space in between two blocks of code.

Reply:
done